os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -275,6 +281,9 @@ void rtl8730_lcdc_initialize(void)
 	struct lcd_dev_s *dev = (struct lcd_dev_s *)mipi_lcdinitialize(dsi_device, &g_rtl8730_config_dev_s.lcd_config);
 	LcdcInitValues(config);
 	rtl8730_lcd_init();
+	rtl8730_enable_lcdc();
+	rtl8730_control_backlight(CONFIG_LCD_MAXPOWER);
```

Comment:
Can we keep backlight power in mipi_lcd.c only. Is there any specific reason to move it here?
If it affects anything, need to add below lines otherwise it will give build error if config is not defined
+#if !defined(CONFIG_LCD_MAXPOWER) || CONFIG_LCD_MAXPOWER < 1
+#define CONFIG_LCD_MAXPOWER 100
+#endif

Reply:
is it related to the tearing issue?

Reply:
@anjana348  yes, control backlight can be kept in mipi_lcd.c, sorry for mistake.
@ewoodev , the rtl8730_enable_lcdc should be done after all lcd initialization operation is completed. Thus, it was moved from mipi_lcd.c to here.